File: mobileBE/src/main/java/com/uth/mobileBE/dto/response/InitialBookResponse.java
package com.uth.mobileBE.dto.response;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class InitialBookResponse {
    private Long bookId;
    private String title;
    private String author;
    private String isbn;
    private Double basePrice;

    private Long libraryId;
    private Long categoryId;

    // thông tin copy đầu tiên
    private Long copyId;
    private String barcode;
    private String condition;
    private String status;
}

File: app/src/main/java/com/example/quanlythuvien/data/model/response/InitialBookResponse.kt
package com.example.quanlythuvien.data.model.response

data class InitialBookResponse(
    val bookId: Long,
    val title: String,
    val author: String,
    val isbn: String,
    val basePrice: Double,
    val libraryId: Long,
    val categoryId: Long,
    val copyId: Long,
    val barcode: String,
    val condition: String,
    val status: String
)